app/routes/task_routes.py

```diff
@@ -1 +1,98 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, Response, abort, make_response
+import requests
+import os
+from .route_utilities import validate_model, create_model
+from datetime import datetime
+from ..models.task import Task
+from ..db import db
+
+bp=Blueprint("tasks_bp", __name__, url_prefix="/tasks")
```

Comment:
Nitpick: spacing is off. 

The assignment operator should have spaces on either side. 

Here's the guidance the PEP8 style guide provides: https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements

```suggestion
bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
```